### PR TITLE
Styled cards

### PR DESCRIPTION
**Pull request by farhaannishtar**
Styling the habit cards in the dashboard.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -88,7 +87,7 @@ export default function Home({ isConnected }) {
             flex-direction: column;
             justify-content: center;
             align-items: center;
-            border: 2px solid purple;
+            /* border: 2px solid purple; */
```
Comment (ahsojar) [inline pages/index.js]: can we remove this?

File: components/Habit.js [hunk 2]
```diff
@@ -94,34 +74,6 @@ export default function Habit(props) {
...
-        .habitInput:hover {
-          background-color: rgba(239,239,239,255);
-        }
-
-        .habitInput::placeholder {
-            font-weight: bold;
-            opacity: 0.25;
-            color: black;
-        }
-
```
Comment (ahsojar) [inline components/Habit.js]: love seeing how much CSS has been deleted 🙌

File: components/InlineEdit.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    if (event.key === "Escape" || event.key === "Enter") event.target.blur();
+  }
+  
+  const onBlur = (event) => {
+    if (event.target.value.trim() === "") {
+      setEditingValue(value);
+    } else {
+      setEditingValue(event.target.value)
+    }
```
Comment (ahsojar) [inline components/InlineEdit.js]: can you explain what these lines are doing?

**Review by ahsojar — APPROVED** [review]
Just reviewed the last commit (97662056f897f9339448ed88214e990e15dd5337) but it looks so much better than what you had yesterday! Both aesthetically, and the code is so much cleaner too.

Love that the InlineEdit is it's own component. Did you find the code for that online, or write it yourself?

Regardless, this is looking a lot better! 

p.s. reminder for future tickets to make a new branch first, then commit new code. Then you can open up a PR against this branch.

I found the code for InlineEdit online. I just changed a few things.

File: components/InlineEdit.js (hunk 3 above)
Comment (farhaannishtar) [reply components/InlineEdit.js:23]: If the input box is empty after being trimmed for whitespace, 

> Just reviewed the last commit ([9766205](https://github.com/farhaannishtar/Habit-Tracker/commit/97662056f897f9339448ed88214e990e15dd5337)) but it looks so much better than what you had yesterday! Both aesthetically, and the code is so much cleaner too.
> 
> Love that the InlineEdit is it's own component. Did you find the code for that online, or write it yourself?
> 
> Regardless, this is looking a lot better!
> 
> p.s. reminder for future tickets to make a new branch first, then commit new code. Then you can open up a PR against this branch.

I found the code for InlineEdit online. I just changed a few things.

Comment (farhaannishtar) [reply components/InlineEdit.js]: After the input box loses focus, (`onBlur`), if the input box is empty after being trimmed for whitespace, don't change the `editingValue` to `""` Change it to what the value is already. I meant to put:

```
if (event.target.value.trim() === "") {
      setEditingValue(editingValue);
    } else {
      setEditingValue(event.target.value)
    }
```

Else, change the editing value to the value of the input box. I'm not sure if that's what it means

**Pull request merged**